src/app/pages/send/choose-crypto-asset/components/crypto-asset-list-item.tsx

```diff
@@ -18,9 +21,18 @@ export function CryptoAssetListItem(props: CryptoAssetListItemProps) {
   const navigate = useNavigate();
   const isBitcoinSendEnabled = useConfigBitcoinSendEnabled();
 
-  function navigateToSendForm(state?: object) {
-    if (blockchain === 'bitcoin' && !isBitcoinSendEnabled) navigate(RouteUrls.SendBtcDisabled);
-    navigate(`${RouteUrls.SendCryptoAsset}/${asset.symbol?.toLowerCase()}`, { state });
+  function navigateToSendForm(isFtToken?: boolean) {
```

Comment:
Small pref but  would prefer as:

```suggestion
  function navigateToSendForm({ isFtToken = false }: { isFtToken: boolean }) {
```
 
or breaking into two fns per
https://github.com/ryanmcdermott/clean-code-javascript#dont-use-flags-as-function-parameters

A readability thing as as when invoking, it isn't clear what true does